src/django/api/tests.py

```diff
@@ -300,10 +300,21 @@ def test_replaces(self):
         self.assertEqual(FacilityList.objects.all().count(),
                          previous_list_count + 2)
         self.assertEqual(new_list.replaces.id, original_list.id)
+        self.assertEqual(new_list.status, FacilityList.PENDING)
+
+        # API should return REPLACED meta-status
+        response = self.client.get(f'/api/facility-lists/{original_list.id}/')
```

Comment:
I couldn't figure out how to reverse this url.

Reply:
Ah, I forgot about `show_urls`. Thanks! https://github.com/open-apparel-registry/open-apparel-registry/pull/2362/commits/dcfb1e0d68fa3d0b096ce3599e9f1b8a47c3ccbe